be/src/types/bitmap_value.h

```diff
@@ -66,7 +66,7 @@ class BitmapValue {
     // Construct an empty bitmap.
     BitmapValue();
 
-    BitmapValue(const BitmapValue& other);
+    BitmapValue(const BitmapValue& other, bool deep_copy = true);
```

Comment:
1. who will use this function?
2. is there any UT cover this function?

Reply:
1. bitmap on cross join will use it, this is the first pr
2. Fixed